src/graphql/client_registry.rs

```diff
@@ -54,11 +54,11 @@ pub async fn query_registry_indexer(
             .first()
             .map(|event| event.indexer.clone())
             .ok_or(QueryError::ParseResponseError(format!(
-                "No indexer data queried from registry for GraphcastID: {graphcast_id_address}"
+                "No indexer data queried from registry for GraphcastID: {wallet_address}"
             )))
     } else {
         Err(QueryError::ParseResponseError(format!(
-            "No response data from registry for graphcastID {graphcast_id_address}"
+            "No response data from registry for graphcastID {wallet_address}"
```

Comment:
There should be a standard through the error messages, one has `:` and they have different formats for `GraphcastID`